vowpalwabbit/options_boost_po.cc

```diff
@@ -111,6 +114,13 @@ void options_boost_po::add_and_parse(const option_group_definition& group)
     po::store(parsed_options, vm);
     po::notify(vm);
   }
+// It seems as though Boost 1.69 was introduced in 1.69 and is later started to be thrown out of Boost PO.
```

Comment:
the comment is a bit confusing

Reply:
wow yeah I messed that up, thanks for pointing it out